### PR TITLE
iox-1391 Minor fixes related to namespace change

### PR DESCRIPTION
**Pull request by mossmaurice**
## Pre-Review Checklist for the PR Author

1. [x] Code follows the coding style of [CONTRIBUTING.md][contributing]
1. [x] ~~Tests follow the [best practice for testing][testing]~~
1. [x] ~~Changelog updated [in the unreleased section][changelog] including API breaking changes~~
1. [x] Branch follows the naming format (`iox-123-this-is-a-branch`)
1. [x] Commits messages are according to this [guideline][commit-guidelines]
    - [x] Commit messages have the issue ID (`iox-#123 commit text`)
    - [x] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
1. [x] Update the PR title
   - Follow the same conventions as for commit messages
   - Link to the relevant issue
1. [x] Relevant issues are linked
1. [x] Add sensible notes for the reviewer
1. [x] All checks have passed (except `task-list-completed`)
1. [x] All touched (C/C++) source code files from `iceoryx_hoofs` are added to `./clang-tidy-diff-scans.txt`
1. [x] Assign PR to reviewer

[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[eca]: http://www.eclipse.org/legal/ECA.php
[contributing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#coding-style
[testing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/concepts/best-practice-for-testing.md
[changelog]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/release-notes/iceoryx-unreleased.md

## Notes for Reviewer

* Fix a few minor left-overs

## Checklist for the PR Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code according to our coding style and naming conventions
- [x] Unit tests have been written for new behavior
- [x] Public API changes are documented via doxygen
- [x] Copyright owner are updated in the changed files
- [x] All touched (C/C++) source code files from `iceoryx_hoofs` have been added to `./clang-tidy-diff-scans.txt`
- [x] PR title describes the changes

## Post-review Checklist for the PR Author

1. [x] All open points are addressed and tracked via issues

## References

- Closes #1391 (partly)

### CODE REVIEW

**Review by dkroenke — DISMISSED** [review]
Only minor fixes, we are good to go here.

File: doc/website/release-notes/iceoryx-v2-0-0.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ servers (`iox::popo::MessagingPattern::REQ_RES`) [\#27](https://github.com/eclip
 - `find_package(iceoryx_posh)` fails [\#944](https://github.com/eclipse-iceoryx/iceoryx/pull/944) thanks to @ijnek
 - iox-roudi report error when running in docker [\#946](https://github.com/eclipse-iceoryx/iceoryx/pull/946)
 - compile error: duration.inl - duration literals are not found (Windows) [\#1078](https://github.com/eclipse-iceoryx/iceoryx/pull/1078)
-- `cxx::string` initialization with nullptr leads to segfault [\#1108](https://github.com/eclipse-iceoryx/iceoryx/pull/1108)
+- `iox::string` initialization with nullptr leads to segfault [\#1108](https://github.com/eclipse-iceoryx/iceoryx/pull/1108)
```
Comment (elBoberido) [inline doc/website/release-notes/iceoryx-v2-0-0.md:66]: Please revert the changes from the v2 release notes :)

Comment (mossmaurice) [reply doc/website/release-notes/iceoryx-v2-0-0.md:66]: Upsi, sure :D

Done :heavy_check_mark:

**Review by elBoberido — APPROVED** [review]

**Review by dkroenke — APPROVED** [review]
